fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
What I'm looking for is something that will return the canonical text for any `ItemKicker`. I.e., “Breaking” for `BreakingNewsKicker`, the tag name for `TagKicker`, and so on. We need plain text, so HTML kickers, if they cannot be made into plain text, should be returned as `None`.